warp10/src/main/java/io/warp10/script/ext/rexec/REXEC.java

```diff
@@ -134,11 +134,8 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
       out.write(WarpScriptLib.TOOPB64.getBytes(StandardCharsets.UTF_8));
       out.write('\n');
       
-      if (this.compress) {
-        out.close();
-      }
-      
-      connout.flush();
+      out.flush();
```

Comment:
The call to flush is not needed since the stream is closed right after.

Reply:
I hope abstract close() comments can be trusted, I didn't check all the close implementation...